charts/device-plugin-operator/templates/operator.yaml

```diff
@@ -462,7 +462,7 @@ spec:
         - --logtostderr=true
         - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
         - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
+        image: "{{ .Values.kube-rbac-proxy.image.hub }}/kube-rbac-proxy:{{ .Values.kube-rbac-proxy.image.tag }}"
```

Comment:
I'm OK with this but there was another ask to get private registries supported too. Did you consider adding that?